manifests/config.pp

```diff
@@ -216,11 +218,13 @@
   file {$client_body_temp_path:
     ensure => directory,
     owner  => $daemon_user,
+    group  => $daemon_group,
   }
 
   file {$proxy_temp_path:
     ensure => directory,
     owner  => $daemon_user,
+    group  => $daemon_group,
```

Comment:
You can remove this. The group is 'root' but with permissions 0644 it will be readable even if you change the daemon user.